### ISSUE TITLE
snowboard/0.6.7: a binary installed via brew doesn't work

### ISSUE THREAD

**Issue by rumyantseva** [bukalapak/snowboard#29]
I use macOS and I was trying to install snowboard via brew.

```
$ brew --version
Homebrew 1.3.0
Homebrew/homebrew-core (git revision fd87; last commit 2017-07-31)

$ brew tap bukalapak/packages
$ brew install snowboard
...
🍺  /usr/local/Cellar/snowboard/0.6.7: 3 files, 25.4MB, built in 3 seconds
```

When I try call snowboard to generate html, I receive an error:

```
$ snowboard html -o output.html handlers.apib
open : no such file or directory

$ which snowboard
/usr/local/bin/snowboard

$ /usr/local/bin/snowboard html -o output.html handlers.apib
open : no such file or directory
```

So, I cloned the repo and used `make install` and after it. I tried snowboard from `$GOPATH/bin` and it works for me.

Perhaps, a version for brew is outdated or something like this.

**Comment by subosito** [comment]
@rumyantseva There are major updates on master for passing input file, which is bit different with snowboard `<= 0.6.7`:

```
# master
$ snowboard html -o output.html handlers.apib

# snowboard <= 0.6.7
$ snowboard html -o output.html -i handlers.apib
```

This major changes will be available as `0.7.0` or `1.0.0`. I am thinking to update error message to be clearer than just `open : no such file or directory`.

Sorry for the confusion.

**Comment by rumyantseva** [comment]
@subosito ok! thank you!
